editor/inserter/button.js

```diff
@@ -3,14 +3,18 @@
  */
 import Inserter from './';
 
-const el = wp.element.createElement;
-
 const InserterButton = ( { opened, onClick } ) => {
-	return el( 'div', { className: 'inserter__button' },
-		el( 'a', { className: 'inserter__button-toggle', onClick },
-			el( 'span', { className: 'dashicons dashicons-plus' } )
-		),
-		opened && el( Inserter )
+	const toggle = ( event ) => {
+		event.preventDefault();
+		onClick();
+	};
+	return (
+		<div className="inserter__button">
+			<a href="" className="inserter__button-toggle" onClick={ toggle }>
```

Comment:
Accessibility rules alert you to problematic node without the `href` so an empty `href` is added. I see through your ruse! 😆 

This should be a `<button>`.